charts/splunk-connect-for-snmp/templates/_helpers.tpl

```diff
@@ -6,6 +6,14 @@
 {{- end }}  
 {{- end }}  
 
+{{- define "splunk-connect-for-snmp.mongodbHost" -}}
+{{- if .Values.mongodbHost }}
```

Comment:
Not sure if this is a good idea, but this option give us option to set URI to something else in case customer's env follows some different pattern than `%s-mongodb.%s.svc.cluster.local"`, not sure if that can even happen. I'll investigate mongodb's in replicaset and ha microk8s